metplus/wrappers/cyclone_plotter_wrapper.py

```diff
@@ -530,9 +549,9 @@ def create_plot(self):
         # ax.legend(loc='lower left', bbox_to_anchor=(-0.03, -0.5),
         #           fancybox=True, shadow=True, scatterpoints=1,
         #           prop={'size': 6})
-        ax.legend(loc='lower left', bbox_to_anchor=(-0.01, -0.5),
-                  fancybox=True, shadow=True, scatterpoints=1,
-                  prop={'size': 6})
+#       ax.legend(loc='lower left', bbox_to_anchor=(-0.01, -0.5),
```

Comment:
Do we need to keep these commented out lines here or should we remove them?

Reply:
I removed the legend because it was too large and obscured the plot.  I just tested with these lines uncommented and the legend is no longer covering the plot area, so I'll uncomment them (they are normally uncommented)

Reply:
These lines were commented out by me because the legend became so large it obscured the figure for feature_728_newTCdata.  Added them back in and tested; the legend no longer obscures the figure.